docs/async.md

```diff
@@ -237,3 +237,44 @@ async with httpx.AsyncClient(transport=transport, base_url="http://testserver")
 ```
 
 See [the ASGI documentation](https://asgi.readthedocs.io/en/latest/specs/www.html#connection-scope) for more details on the `client` and `root_path` keys.
+
+## Startup/shutdown of ASGI apps
+
+It is not in the scope of HTTPX to trigger lifespan events of your app.
+
+However it is suggested to use `LifespanManager` from [asgi-lifespan](https://github.com/florimondmanca/asgi-lifespan#usage) in pair with `AsyncClient`.
```

Comment:
Given that the examples below don't include any extra asgi-lifespan/HTTPX integration hints, should we drop everything below this line I'm commenting on? So as to avoid duplicate information. People can just click on the link and learn what `asgi-lifespan` is about and how to use it. There's even an HTTPX example in the README there. :)

Reply:
SGTM 👍